src/main.rs

```diff
@@ -89,6 +113,9 @@ impl MessageEvent<WorkerParameters> for TranscriptEvent {
     let selected_streams = get_first_audio_stream_id(&format_context)?;
 
     let cloned_parameters = parameters;
+    let param_output_format = cloned_parameters.output_format.clone();
+
+    let output_format = OutputFormat::from_str(&(param_output_format.unwrap_or("JSON".to_string()))).unwrap();
```

Comment:
So the  default output format is JSON here but EBU_TT_D at line [81](https://github.com/media-cloud-ai/rs_transcript_worker/pull/6/files#diff-42cb6807ad74b3e201c5a7ca98b911c5fa08380e942be6e4ac5807f8377f87fcR81)?

Reply:
You should also handle the error instead of "unwrap()"